src/osd/PrimaryLogPG.cc

```diff
@@ -2517,6 +2517,32 @@ void PrimaryLogPG::do_op(OpRequestRef& op)
   maybe_force_recovery();
 }
 
+bool PrimaryLogPG::need_skip_handle_manifest(const OpRequestRef& op_req,
+                                             const bool is_chunked)
```

Comment:
pls remove the 'const' from the 2nd argument

Reply:
Okay. Thanks. I'll fix it.